tasks/system_probe.py

```diff
@@ -590,13 +590,10 @@ def build_security_ebpf_files(ctx, build_dir):
 
 def build_bcc_files(ctx, build_dir):
     corechecks_c_dir = os.path.join(".", "pkg", "collector", "corechecks", "ebpf", "c")
-    corechecks_bcc_dir = os.path.join(corechecks_c_dir, "bcc")
     bcc_files = [
-        os.path.join(corechecks_bcc_dir, "tcp-queue-length-kern.c"),
-        os.path.join(corechecks_c_dir, "tcp-queue-length-kern-user.h"),
-        os.path.join(corechecks_bcc_dir, "oom-kill-kern.c"),
+        os.path.join(corechecks_c_dir, "oom-kill-kern.c"),
         os.path.join(corechecks_c_dir, "oom-kill-kern-user.h"),
-        os.path.join(corechecks_bcc_dir, "bpf-common.h"),
+        os.path.join(corechecks_c_dir, "bpf-common.h"),
```

Comment:
This is no longer the correct thing to do for building these probes. See https://github.com/DataDog/datadog-agent/blob/8f141e218ee5bd74d6a7742e73844220c5cda603/tasks/system_probe.py#L636